packages/manager/src/components/TextTooltip/TextTooltip.tsx

```diff
@@ -29,33 +28,21 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-const akamaiBillingInvoiceText = (
-  <Typography>
-    Charges in the final Akamai invoice should be considered the final source
-    truth. Linode invoice will not reflect discounting, currency adjustment, or
-    any negotiated terms and conditions. Condensed and finalized invoice is
-    available within{' '}
-    <Link to="https://control.akamai.com/apps/billing">
-      Akamai Control Center &gt; Billing <ExternalLinkIcon />
-    </Link>
-    .
-  </Typography>
-);
-
-export const BillingTooltip: React.FC<Props> = (props) => {
+export const TextTooltip: React.FC<Props> = (props: Props) => {
```

Comment:
Can we remove React.FC ?
```suggestion
export const TextTooltip = (props: Props) => {
```